test/integration/roles/setup_selinux/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+- name: Install packages for selinux
+  package:
+    name: libselinux-python
+    state: present
```

Comment:
Do we need to install the libselinux-python3 package if ansible is running on python3?